files-community/Files
# Feature: Added support for local branch checkout

**Resolved / Related Issues**
   Related #11014 

**Notes**
- The flyout is clipped because of #11713
- During the checkout the layout is updated more than once, I want to find a way to avoid that

**Validation**
How did you test these changes?
   1. Open folder with git repo with uncommitted changes
   2. Try switching branches from the status bar flyout
   3. Try all the available options

**Screenshots (optional)**
![image](https://github.com/files-community/Files/assets/39923744/d2fc53f3-18c2-4d38-a022-e4cef5e9cc94)
![1](https://github.com/files-community/Files/assets/102259289/96c9dffa-f45b-4e1c-95ec-386bf3699b1f)
